src/api/publications.ts

```diff
@@ -11,3 +20,25 @@ export const createPublication = (
         detail: entityDescription ?? null,
     });
 };
+
+export const getPublicationById = async (pubId: string) => {
+    const data = await supabaseRetry(
+        () =>
+            supabaseClient
+                .from(TABLES.PUBLICATIONS)
+                .select(JOB_STATUS_COLUMNS)
+                .eq('id', pubId),
```

Comment:
We are now duplicating this query 3 times. If I remember right it might be a bit of a pain to get all three of these shared. Please look into it though.